convert_to_excel.py

```diff
@@ -56,53 +62,70 @@
 from openpyxl.styles import Font
 from openpyxl.styles import Alignment
 from openpyxl.styles import PatternFill
-
-print('Start\n')  # Only for testing purposes---.
-
+import sys
+import os
 ##############################################################################
-""" Variables, lists, etc. """
 
 
-column_names = ['Task', 'Set Part', 'Parent Build', 'Start Date', 'End Date']
-corresponding_keys = ['content', 'entity', 'sg_parent_build',
-                      'start_date', 'due_date']
-
-imported_data = 'test_data.pkl'
-workbook_title = 'converted_data.xlsx'
-worksheet_title = 'Formatted'
-
 
 ##############################################################################
-""" 1. Unpickle file and place into a list. """
-
-# Convert pickled data into a list.
-unpickled_data = open(imported_data, 'rb')
-data = pickle.load(unpickled_data)
-# print(data)  # [works] This is a list, not a dict.
-
-# Check type of the converted data.
-# t = type(data)  # Only for testing purposes---.
-# print('type = {}\n'.format(t))  # Only for testing purposes---.
+""" 1. Defining the data class. """
+
+
+class CapturedData:
+    def __init__(self, object_name, task_name, set_part,
+                 parent_build, start_date,
+                 end_date):
+
+        self.object_name = object_name
+        self.task_name = task_name
+        self.set_part = set_part
+        self.parent_build = parent_build
+        self.start_date = start_date
+        self.end_date = end_date
+
+    def print_status(self):
```

Comment:
Rather than using this `print_status` function I would recommend utilizing the python built in method `__str__` here. That will allow you to just use `print(my_captured_data)` which is more pythonic than `my_captured_data.print_status()`